### PR TITLE
fix #241 - `body` should be supported as identifier

### LINKED ISSUE #241: `body` should be supported as identifier

### PR DESCRIPTION
**Pull request by BBasile**

### CODE REVIEW

File: test/tester.d [hunk 1]
```diff
@@ -295,7 +295,7 @@ int main(string[] args)
         config.fileName = arg;
         const(Token)[] tokens = getTokensForParser(fileBytes, config, &cache);
         RollbackAllocator rba;
-        parseModule(tokens, arg, &rba, &messageFunction);
+        parseModule(tokens, arg, &rba, toDelegate(&messageFunction));
```
Comment (Geod24) [inline test/tester.d]: That looks unrelated, isn't it ?

Comment (BBasile) [reply test/tester.d]: indeed.

Comment (BBasile) [reply test/tester.d]: removed
